contracts/test/Tub.sol

```diff
@@ -28,6 +29,29 @@ contract Tub is Ownable {
         emit Store(_value);
     }
 
+    function multiStore(string[] calldata boxValues) external {
+        for (uint256 i = 0; i < boxValues.length; i++) {
+            _value = boxValues[i];
+            emit Store(_value);
+        }
+    }
+
+    function nestedStore(string calldata boxValue, address[] calldata nested)
+        external
+    {
+        _value = boxValue;
+
+        emit Store(_value);
+
+        if (nested.length > 0) {
+            emit MultiStore(boxValue, nested);
+            Tub(nested[0]).nestedStore(
+                string(abi.encodePacked("+", boxValue)),
+                nested[1:]
+            ); // string(abi.encodePacked("+", boxValue))
```

Comment:
nit; commented out code

Reply:
removed already - it wasnt a finished version ;)